runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -427,7 +429,13 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
       J9ROMClass *declaringClazz = J9_CLASS_FROM_METHOD(method)->romClass;
       J9UTF8 * className = J9ROMCLASS_CLASSNAME(declaringClazz);
       J9UTF8 * name = J9ROMMETHOD_NAME(romMethod);
-      if (J9UTF8_LENGTH(className) == 36
+      if (highCodeCacheOccupancyThresholdReached && !countInOptionSet && !TR::Options::getCountsAreProvidedByUser())
+         {
+         count = J9ROMMETHOD_HAS_BACKWARDS_BRANCHES(romMethod) ?
+                     TR::Options::getHighCodeCacheOccupancyBCount() :
+                     TR::Options::getHighCodeCacheOccupancyCount();
+         }
+      else if (J9UTF8_LENGTH(className) == 36
```

Comment:
Those special methods always need to have count=0 so that they get jit compiled before the first invocation. Thus let's put our code after these two special cases